sklearn/discriminant_analysis.py

```diff
@@ -623,11 +623,15 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Discriminant Analysis
     """
 
-    def __init__(self, priors=None, reg_param=0., store_covariances=False,
-                 tol=1.0e-4):
+    def __init__(self, priors=None, reg_param=0.,store_covariance=False, tol=1.0e-4,store_covariances=None):
```

Comment:
@chinmay0301 Just be calm, I'll guide you through the process. 
Firstly, change this to:
```python
def __init__(self, priors=None, reg_param=0., store_covariance=False,
             tol=1.0e-4, store_covariances=False):
```
This differs from the one you wrote because this follows the PEP8 guidelines. Please go throught the Contributor's guide for more information.